marathon-bigip-ctlr.py

```diff
@@ -314,8 +314,8 @@ def get_source_addr_translation(source_addr_translation):
     This configures virtual server source address translation based on app
     label or from default.
     """
-    if str(source_addr_translation) == 'automap':
-        return {'type': 'automap'}
+    if source_addr_translation == {'type': 'automap'}:
```

Comment:
I'm not sure why this special check is now needed.  Can this function be called with both a json formatted string as well as a json object?  I only see one call in the controller code and it seems to be a json formatted string.

Reply:
This check needs to be here for the default case when that label has not been set so the default app.source_addr_translation = {'type': 'automap'} instead of a json encoded string.